### PR TITLE
Add test for screen wake lock on worker

### PR DESCRIPTION
**Pull request by Honry**
which should not be allowed

### CODE REVIEW

**Comment by Honry** [comment]
@rakuco, @kenchris, please also help review this one. Thanks! :)

File: wake-lock/wakelock-screen-type-on-worker.https.worker.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//META: title=Screen wake lock should not be allowed in dedicated worker
+importScripts("/resources/testharness.js");
+
+promise_test(async t => {
+  await promise_rejects(t, "NotAllowedError", WakeLock.request('screen'));
```
Comment (rakuco) [inline wake-lock/wakelock-screen-type-on-worker.https.worker.js]: I think `async` and `await` are unnecessary here: simply `return`ing  `promise_rejects()` also works.

**Review by marcoscaceres — APPROVED** [review]

**Comment by marcoscaceres** [comment]
Approving, but please make the changes suggested by @rakuco 🙏

File: wake-lock/wakelock-screen-type-on-worker.https.worker.js (hunk 1 above)
Comment (Honry) [reply wake-lock/wakelock-screen-type-on-worker.https.worker.js]: Done. :)

**Pull request merged**